src/main/java/com/uber/cadence/internal/sync/TestWorkflowEnvironmentInternal.java

```diff
@@ -17,7 +17,76 @@
 
 package com.uber.cadence.internal.sync;
 
-import com.uber.cadence.*;
```

Comment:
Any reason why this as expanded? Or is this auto-formated?

Reply:
The convention is to always expand the imports, so ppl can check what exactly is imported. That said the IDE sometimes automatically collapses the import and that's why you see starts in some place.